files/Dockerfile

```diff
@@ -145,37 +137,112 @@ RUN echo "Install/build apps that are not provided by the OS packages." && \
-    # tmux && \
-        curl -sSL https://github.com/libevent/libevent/releases/download/release-2.0.22-stable/libevent-2.0.22-stable.tar.gz | tar xz && \
-        cd libevent-* && \
-        ./configure && make &&  make install && \
-        cd .. && rm -fr libevent-* && \
-        cd /tmp && curl -sSL https://github.com/tmux/tmux/releases/download/2.7/tmux-2.7.tar.gz | tar xz && \
-        cd tmux-* && \
-        ./configure --prefix=/usr/local && \
-        make &&  make install && \
-        cd .. && rm -fr tmux-* && \
+        curl -sSL https://github.com/drwetter/testssl.sh/archive/v2.9.5-7.tar.gz | tar xz && \
+        mv testssl* /usr/local/share/testssl && ln -s /usr/local/share/testssl/testssl.sh /usr/local/bin/testssl && chmod a+x /usr/local/bin/testssl && \
+    # yadm && \
+        curl -sSL https://github.com/TheLocehiliosan/yadm/archive/1.12.0.tar.gz | tar xz && \
+        mv yadm* /usr/local/share/yadm && ln -s /usr/local/share/yadm/yadm /usr/local/bin/yadm && chmod a+x /usr/local/bin/yadm
+
+
+#######################################
+# Configure systemd and misc configuration bits
+#  Mostly taken from:
+#  https://github.com/dramaturg/docker-debian-systemd/blob/master/Dockerfile
+
+RUN cd /lib/systemd/system/sysinit.target.wants/ && \
+        ls | grep -v systemd-tmpfiles-setup.service | xargs rm -f && \
+        rm -f /lib/systemd/system/sockets.target.wants/*udev* && \
+        systemctl mask -- \
+            tmp.mount \
+            etc-hostname.mount \
+            etc-hosts.mount \
+            etc-resolv.conf.mount \
+            -.mount \
+            swap.target \
+            getty.target \
+            getty-static.service \
+            dev-mqueue.mount \
+            cgproxy.service \
+            systemd-tmpfiles-setup-dev.service \
+            systemd-remount-fs.service \
+            systemd-ask-password-wall.path \
+            systemd-logind.service && \
+                systemctl mask -- \
+                        cron.service \
+                        dbus.service \
+                        exim4.service \
+                        ntp.service && \
+        systemctl set-default multi-user.target || true && \
+        sed -ri /etc/systemd/journald.conf -e 's!^#?Storage=.*!Storage=volatile!' && \
+        # Set locale && \
+        localedef -i en_US -f UTF-8 en_US.UTF-8 && \
+        # Configure openssh-server && \
+        sed -i 's/#Port 22/Port 2022/' /etc/ssh/sshd_config && \
+        # Configure docker daemon to support docker in docker && \
+        mkdir /etc/docker && echo '{"storage-driver": "vfs"}' > /etc/docker/daemon.json
+
+#######################################
+# Install python-based utils and libs
+RUN echo "Install python-based utils and libs" && \
+    curl -sSL https://bootstrap.pypa.io/get-pip.py | python2 && \
+    pip2.7 install -U setuptools && \
+    pip2.7 install \
```

Comment:
I think This may not be necessary to embed this rather than pull from files.



For the ADD and COPY instructions, the contents of the file(s) in the image are examined and a checksum is calculated for each file. The last-modified and last-accessed times of the file(s) are not considered in these checksums. During the cache lookup, the checksum is compared against the checksum in the existing images. If anything has changed in the file(s), such as the contents and metadata, then the cache is invalidated.